js/main.js

```diff
@@ -4,36 +4,56 @@ const addBtn = document.querySelector('#addBtn');
 const title = document.querySelector('#title');
 const arthur = document.querySelector('#arthur');
 let bookObj = [];
+const addNewTab = document.querySelector('#addNew')
+const listTab = document.querySelector('#list')
+const contactTab = document.querySelector('#contact')
+const header = document.querySelector('#header')
+header.addEventListener('click',function(e) {
+if(e.target.id === 'listTab') {
+ addNewTab.classList.add('removeDisplay')
+ contactTab.classList.add('removeDisplay')
+ listTab.classList.remove('removeDisplay')
+}
+else if(e.target.id === 'addNewTab') {
+  addNewTab.classList.remove('removeDisplay')
+  listTab.classList.add('removeDisplay')
+ contactTab.classList.add('removeDisplay')
+}
+else if(e.target.id === 'contactTab') {
+  contactTab.classList.remove('removeDisplay')
+  listTab.classList.add('removeDisplay')
+ addNewTab.classList.add('removeDisplay')
+}
+})
 class Books {
-  /* eslint-disable */
-  constructor(bookTitle, arthur, id) {
+  constructor(bookTitle,arthur,id) {
     this.bookTitle = bookTitle,
     this.arthur = arthur,
-    this.id = id;
+    this.id = id
   }
 }
 class BookObj {
-  static loop() {
+ static  loop() {
     let objVar = '';
     for (let i = 0; i < bookObj.length; i += 1) {
       objVar += `<li id='lanzz'><p><span>${bookObj[i].bookTitle}</span> by <span>${bookObj[i].arthur}</span> </p> <button onclick = 'BookObj.removeBook(this.id)' id=${bookObj[i].id.toString()}>Remove</button></li>`;
     }
     bookList.innerHTML = objVar;
   }
 
-  static removeBook(id) {
-    filteredArr = bookObj.filter((item) => item.id !== id);
-    bookObj = filteredArr;
-    BookObj.loop();
-    localStorage.setItem('inputArr', JSON.stringify(bookObj));
-  }
+  static  removeBook(id) {
+     filteredArr = bookObj.filter((item) => item.id !== id);
+     bookObj = filteredArr;
+     BookObj.loop();
+     localStorage.setItem('inputArr', JSON.stringify(bookObj));
+   }
 }
-
+ 
 addBtn.addEventListener('click', () => {
   if (title.value) {
     const titleValue = title.value;
     const arthurValue = arthur.value;
-    const newBook = new Books(titleValue, arthurValue);
+    const newBook = new Books(titleValue,arthurValue)
     bookObj.push(newBook);
     let objVar = '';
     for (let i = 0; i < bookObj.length; i += 1) {
```

Comment:
- [ ] I noticed on adding a book, the input does not clear but remains on the screen. Though the book gets added, it will be a great user experience to have to clear up for a user to know it added up

## View:
![image](https://user-images.githubusercontent.com/50990925/218998940-6844c4ee-be18-40f3-8c2b-5245b763e417.png)